server/monitor.go

```diff
@@ -123,6 +123,7 @@ type ConnInfo struct {
 	OutMsgs        int64          `json:"out_msgs"`
 	InBytes        int64          `json:"in_bytes"`
 	OutBytes       int64          `json:"out_bytes"`
+	SentBytes      int64          `json:"sent_bytes"`
```

Comment:
Should we not have ReceivedBytes as well if other side sending us compressed?
I do not like long winded but should it be SentWireBytes?

Reply:
The problem with the receive side is that we would only get the bytes read *after* decompression (since we have a s2 reader that wraps the net.Conn).
As for `SentBytes`, I originally used `WriteBytes`, but decided on `SentBytes`. I can certainly change to `SentWireBytes`. Please let me know and I will make the change.

Reply:
(edited, I wrote `WriteBytes` when I meant that I originally used `WireBytes`)

Reply:
I would remove for now, we need better and more symmetric reporting long term but not here and I want us to think a bit more about it.

Reply:
will do